rapidast.py

```diff
@@ -68,15 +85,8 @@ def load_environment():
     # Update to latest config schema if need be
     config = configmodel.converter.update_to_latest_config(config)
 
-    # Choose where the results are going to be stored.
-    # Enforced result layout: {config.results_base_dir}/{application.shortName}/{<date-time>}/<scanner-name>
-    # This way each runs will get their own directory, and each scanner their own subdir
     scan_time_str = datetime.now().strftime("%Y%m%d-%H%M%S")
-    results_dir = os.path.join(
-        config.get("config.base_results_dir", default="./results"),
-        config.get("application.shortName", default="scannedApp"),
-        scan_time_str,
-    )
+    results_dir = get_full_result_dir_path(config, scan_time_str)
```

Comment:
"how about removing scan_time_str, and creating it directly in get_full_result_dir_path() " => The reason I am doing this is to test the function in the pytest testcase. I wonder if there is another way of testing it.

RE: results_dir, yes it can be removed.